system/index.js

```diff
@@ -30,54 +48,77 @@ module.exports = async (message) => {
   const payload = message.payload;
   // Remap headers to a standard JS object
   const headers = message.headers.toRiffHeaders();
-  Object.keys(headers).map((key) => {headers[key] = message.headers.getValue(key)});
-  if (DEBUG) {
-    console.log('==System Function Start==');
-    console.log(`HEADERS: ${JSON.stringify(headers)}`);
-    console.log(`PAYLOAD: ${JSON.stringify(payload)}`);
-    console.log(`MIDDLEWARE_FUNCTION_URI: ${MIDDLEWARE_FUNCTION_URI}`);
-    console.log('==Middleware Function(s) Start==');
-  }
+  Object.keys(headers).map((key) => { headers[key] = message.headers.getValue(key) });
+  const requestID = headers['x-request-id']
+
+  const logLevel = DEBUG ? 'debug' : 'info';
+  const log = initLogging(logLevel, requestID);
+
+  log.debug({
+    headers: JSON.stringify(headers),
+    payload: JSON.stringify(payload),
```

Comment:
We probably want to mask GDPR stuff.  For SF originating requests, that includes userId (username?).  Then there's the customer payload which may have personal data.  In core, we opted to just log the size of the payload instead of attempt to guess what to mask.